src/workspace-data/Data.js

```diff
@@ -341,7 +342,8 @@ const DataTableActions = ({
             },
             [input({ type: 'hidden', name: 'FCtoken', value: getTerraUser().token }), input({ type: 'hidden', name: 'model', value: 'flexible' })]
           ),
-          (dataProvider.features.supportsTsvDownload || dataProvider.features.supportsTsvAjaxDownload) &&
+          dataProvider &&
```

Comment:
These checks haven't been necessary, but they were useful during development and may be needed in future PRs